### PR TITLE
improve our kubernetes example documentation

### PR DESCRIPTION
**Pull request by garypen**
and add support for prometheus metrics to our helm chart

I deployed a release of the router into our testing cluster using helm and ensured that I could read the prometheus statistics easily from the deployed router.

fixes: #958

### CODE REVIEW

File: docs/source/containerization/kubernetes.mdx [hunk 1]
```diff
@@ -5,9 +5,206 @@ description: Using router images with kubernetes
...
+[Helm](https://helm.sh) is the package manager for kubernetes.
+
+There is a complete [helm chart definition](https://github.com/apollographql/router/tree/main/helm/chart/router) in the repo which illustrates how to use helm to deploy the router in kubernetes.
+
+Here's an example which would use helm to install the router:
+ - into namespace "router-deploy" (create namespace if it doesn't exist)
+ - with helm install name "router-test"
+ - with support for prometheus enabled
+
+You would run this command from <repo>/helm/chart directory.
```
Comment (BrynCooke) [inline docs/source/containerization/kubernetes.mdx:21]: `<repo>` is causing the docs to fail

Comment (garypen) [reply docs/source/containerization/kubernetes.mdx]: fixed in: 247d6f1a36bf8297acbfccc820f09c9c85517a15

**Review by BrynCooke — APPROVED** [review]
Approved but now needs updating to `rc.0`

**Comment by garypen** [comment]
> Approved but now needs updating to `rc.0`

done: 33e4baf08c92dd731338f3ef5728dedc9829d23a

**Review by bnjjj — APPROVED** [review]

**Pull request merged**